index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Definitions by: ricardo-devis-agullo <https://github.com/ricardo-devis-agullo>
```

Comment:
Would you mind removing this? The attribution will stay forever in the git commit log, as every other change in this repo.